admiral/pkg/controller/admiral/dependency_test.go

```diff
@@ -20,3 +23,76 @@ func TestNewDependencyController(t *testing.T) {
 		t.Errorf("Dependency controller should never be nil without an error thrown")
 	}
 }
+
+func TestGetDependencies(t *testing.T) {
+	stop := make(chan struct{})
+	handler := test.MockDependencyHandler{}
+
+	dependencyController, err := NewDependencyController(stop, &handler, "../../test/resources/admins@fake-cluster.k8s.local", "ns", time.Duration(1000))
```

Comment:
You can use `Common.GetKubeconfigPath()` here - we already have a fake config loaded in for testing

Reply:
This doesn't work unless every package has `testdata/fakeconfig` as the file path is relative.